src/main/java/picard/analysis/GcBiasMetricsCollector.java

```diff
@@ -178,11 +225,8 @@ private double sum(final int[] values) {
             return total;
         }
 
-        /////////////////////////////////////////////////////////////////////////////
-        //Called to add metrics to the output file for each level of collection
-        // these metrics are used for graphing gc bias in R script
-        /////////////////////////////////////////////////////////////////////////////
-        public void addMetricsToFile(final MetricsFile<GcBiasMetrics, Integer> file) {
+        private void addGcDataToFile(final MetricsFile<GcBiasMetrics, Integer> file, final Map<String, GcObject> gcData,
```

Comment:
OK. I think that it could be made static, but concede that it would be beyond the scope of this PR.